src/DynamoCore/Core/CustomNodeDefinition.cs

```diff
@@ -230,7 +230,7 @@ public string MangledName
     /// </summary>
     public class CustomNodeInfo
     {
-        public CustomNodeInfo(Guid functionId, string name, string category, string description, string path)
+        public CustomNodeInfo(Guid functionId, string name, string category, string description, string path, bool isVisibleInDynamoLibrary)
```

Comment:
You can assign a default value to the last parameter, so you won't have to update the callers.
```
public CustomNodeInfo(Guid functionId, string name, string category, string description, string path, bool isVisibleInDynamoLibrary = true)
```